CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ cmake_minimum_required(VERSION 3.3)
 
 find_package(WPEFramework)
 
+if(HAVE_ERM)
```

Comment:
Pls move this to services.cmake. Also pls rename it to something like BUILD_ENABLE_ERM to match similar build flags.